client/my-sites/stats/features/modules/stats-top-posts/stats-top-posts.tsx

```diff
@@ -47,10 +46,9 @@ const StatsTopPosts: React.FC< StatsModulePostsProps > = ( {
 	const siteId = useSelector( getSelectedSiteId ) as number;
 
 	const isArchiveBreakdownEnabled: boolean = config.isEnabled( 'stats/archive-breakdown' );
-	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
-	const supportUrl = isOdysseyStats
-		? `${ JETPACK_SUPPORT_URL_TRAFFIC }#analyzing-popular-posts-and-pages`
-		: TOP_POSTS_SUPPORT_URL;
+	const supportContext = config.isEnabled( 'is_running_in_jetpack_site' )
+		? 'stats-top-posts-and-pages-analyze-content-performance-jetpack'
+		: 'stats-top-posts-and-pages-analyze-content-performance';
```

Comment:
Why didn't you change this part like the rest of the PR to use the `isJetpackSite` selector?

Reply:
huh. I had a conflict on this line and I guess I botched this condition.